build/CNAME

```diff
@@ -1 +0,0 @@
-immigrantpoa.org
```

Comment:
Is there a reason we re removing this?

Reply:
I think it should be in the root of the repo. https://help.github.com/articles/troubleshooting-custom-domains/